tracer/src/Datadog.Trace/TracerManagerFactory.cs

```diff
@@ -261,7 +261,7 @@ protected virtual ITraceSampler GetSampler(TracerSettings settings)
             // Note: the order that rules are registered is important, as they are evaluated in order.
             // The first rule that matches will be used to determine the sampling rate.
 
-            if (settings.AppsecStandaloneEnabledInternal)
```

Comment:
nit: I think we should add a comment here. Without the reference to AppSecStandalone mode, it's not at all clear why we need a sampler _at all_ if tracing is disabled